src/reducers/platform.test.js

```diff
@@ -75,55 +76,56 @@ describe('app reducer', () => {
 });
 
 describe('api reducer', () => {
-	beforeEach(function() {
-		this.MOCK_STATE = { foo: 'bar' };
-	});
 	it('returns default state for empty action', () => {
-		expect(api({ ...DEFAULT_APP_STATE }, {})).toEqual(DEFAULT_APP_STATE);
-	});
-	it('re-sets api state on logout API_REQ, with isFetching:true', function() {
-		const logoutRequest = apiActions.requestAll([], { logout: true });
-		expect(api(this.MOCK_STATE, logoutRequest)).toEqual({
-			...DEFAULT_APP_STATE,
-			isFetching: true,
+		expect(api({ ...DEFAULT_API_STATE }, {})).toEqual(DEFAULT_API_STATE);
+	});
+	it('re-sets api state on logout API_REQ, with inFlight query', function() {
+		const ref = 'foobar';
+		const logoutRequest = apiActions.requestAll([{ ref }], { logout: true });
+		expect(api({ ...DEFAULT_API_STATE }, logoutRequest)).toEqual({
+			...DEFAULT_API_STATE,
+			inFlight: [ref],
 		});
 	});
 	it('adds success response to state tree', () => {
 		const API_RESP_SUCCESS = apiActions.success({ query: {}, response: { ref: 'bing', value: 'baz' } });
-		expect(api({ foo: 'bar'}, API_RESP_SUCCESS)).toEqual({
+		expect(api({ ...DEFAULT_API_STATE, foo: 'bar'}, API_RESP_SUCCESS)).toEqual({
 			foo: 'bar',
-			bing: { value: 'baz' }
+			bing: { ref: 'bing', value: 'baz' },
+			inFlight: [],
 		});
 	});
 	it('adds error response to state tree', () => {
-		const API_RESP_ERROR = apiActions.error({ response: { ref: 'bing', bar: 'baz' } });
-		expect(api({ foo: 'bar'}, API_RESP_ERROR)).toEqual({
+		const API_RESP_ERROR = apiActions.error({ response: { ref: 'bing', error: 'baz' } });
+		expect(api({ ...DEFAULT_API_STATE, foo: 'bar'}, API_RESP_ERROR)).toEqual({
 			foo: 'bar',
-			bing: { bar: 'baz' },
+			bing: { ref: 'bing', error: 'baz' },
+			inFlight: [],
 		});
 	});
 	it('populates an `fail` key on API_RESP_FAIL', () => {
 		const API_RESP_FAIL = apiActions.fail(new Error('this is the worst'));
-		const errorState = api({ ...DEFAULT_APP_STATE }, API_RESP_FAIL);
+		const errorState = api({ ...DEFAULT_API_STATE }, API_RESP_FAIL);
 		expect(errorState).toEqual(expect.objectContaining({ fail: API_RESP_FAIL.payload }));
 	});
-	it('sets isFetching:true on API_REQ', () => {
```

Comment:
the deprecated tests are for the `app reducer` starting on line 13 - the changes here are just to update the tests to use a DEFAULT_STATE specific to `state.api` and change from `isFetching` to `inFlight` in the new system.